iron-icon.html

```diff
@@ -52,7 +52,7 @@
 
     <iron-icon icon="fruit:cherry"></iron-icon>
 
-See [iron-iconset](#iron-iconset) and [iron-iconset-svg](#iron-iconset-svg) for more information about
+See [iron-iconset](iron-iconset) and [iron-iconset-svg](iron-iconset-svg) for more information about
 how to create a custom iconset.
 
 See [iron-icons](https://elements.polymer-project.org/elements/iron-icons?view=demo:demo/index.html) for the default set of icons.
```

Comment:
hmm good question do you think linking to iron-icons would be better than iron-iconset && iron-iconset-svg for guides on how to create a custom  iconset?

I was +1 on this PR just because the  current links are broken :boom: 

:violin: